flixel/tweens/FlxEase.hx

```diff
@@ -151,25 +155,60 @@ class FlxEase
 
 	public static function bounceIn(t:Float):Float
 	{
-		return 1 - bounceOut(1 - t);
+		return BOUNCE_IN(t);
+	}
+
+	public static function bounceInCustom(/*bounces:Int = 6, */?Bounce1:Float, ?Bounce2:Float, ?Bounce3:Float, ?Bounce4:Float, ?Bounce5:Float, ?Bounce6:Float):EaseFunction
```

Comment:
these need doc comments.

how are the args determined? right now they seem quite arbitrary, and only determined via long trial and error. what do these values mean to the dev using this function?

Reply:
furthermore, can you provide for me an example of a good custom bounce function that I may try